src/components/learner-credit-management/SubBudgetCard.jsx

```diff
@@ -109,7 +106,7 @@ const BaseSubBudgetCard = ({
       to={`/${enterpriseSlug}/admin/${ROUTE_NAMES.learnerCredit}/${budgetId}`}
       variant={[BUDGET_STATUSES.expired, BUDGET_STATUSES.retired].includes(budgetLabel.status) ? 'outline-primary' : 'primary'}
```

Comment:
nit: `budgetLabel.status` is used here, despite just `status` being used elsewhere and below. Should this line also update to use `status` directly, too?